webui/src/styles/globals.css

```diff
@@ -9,6 +9,19 @@
   --color-bg-conflict: #fff3cd;
 }
 
+.gray-out {
+  opacity: 50%;
```

Comment:
Alternatively, you could use `rgba(255, 255, 255, 0.5)`. The difference is that `opacity` is applied to all children and `rgba` is applied only to the element matching the selector. Since the grayed out div is a sibling to the actual content, it wouldn't make much of a difference in this case, but FYI